store/postgres/src/relational_queries.rs

```diff
@@ -2182,7 +2186,11 @@ impl<'a> SortKey<'a> {
                 out.push_identifier(name)?;
                 out.push_sql(", to_tsquery(");
 
-                out.push_bind_param::<Text, _>(&value.unwrap())?;
+                let mut value = value.unwrap().to_string();
+                if value.contains(" ") {
+                    value = format!("'{}'", value);
+                }
+                out.push_bind_param::<Text, _>(&value.as_str())?;
```

Comment:
Since the exact same code is used twice, it would be better to factor that into a helper `fn to_tsquery(value: &str, out: &mut AstPass<Pg>) -> QueryResult<()> { .. }`

Reply:
there are a few more edges like the List case which does a similar thing, I was thinking that this needs a little more work. I was wondering if it would be worth adding something like a quote fn for handling this in all the places as well as add the tests.